src/Theme_Mod_Command.php

```diff
@@ -121,6 +121,47 @@ public function get( $args = array(), $assoc_args = array() ) {
 
 	}
 
+	/**
+	 * Gets a list of theme mods.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--field=<field>]
+	 * : Returns the value of a single field.
+	 *
+	 * [--format=<format>]
+	 * : Render output in a particular format.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - json
+	 *   - csv
+	 *   - yaml
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Gets a list of theme mods.
+	 *     $ wp theme mod list
+	 *     +------------------+---------+
+	 *     | key              | value   |
+	 *     +------------------+---------+
+	 *     | background_color | dd3333  |
+	 *     | link_color       | #dd9933 |
+	 *     | main_text_color  | #8224e3 |
+	 *     +------------------+---------+
+	 *
+	 * @subcommand list
+	 */
+	public function list_( $args = array(), $assoc_args = array() ) {
+		if ( ! array_key_exists( 'all', $assoc_args ) ) {
```

Comment:
Now that I think about this some more, I'd prefer to have it just always forcefully set `$assoc_args['all']` to `1` here. If you check for existence of the key first, a user could do something like this, which would produce an error:

```
wp theme mod list --no-all
```

So please just remove the `if ( ! array_key_exists( ... ) { }` that wraps the assignment.